rundetection/rules/sans_rules.py

```diff
@@ -121,57 +106,104 @@ def _set_transmission_file(job_request: JobRequest, sample_title: str, sans_file
         if can_scatter is not None:
             job_request.additional_values["can_scatter"] = can_scatter.run_number
 
         # If using M4 monitor then can scatter is the transmission
         if not job_request.additional_values["included_trans_as_scatter"]:
             can_trans = _find_can_trans_file(sans_files=sans_files, can_title=can_title)
-            logger.info("LOQ can trans found %s", can_trans)
+            logger.info("%s can trans found %s", job_request.instrument, can_trans)
         else:
             can_trans = can_scatter
-            logger.info("LOQ can trans set as scatter %s", can_scatter)
+            logger.info("%s can trans set as scatter %s", job_request.instrument, can_scatter)
         if can_trans is not None and can_scatter is not None:
             job_request.additional_values["can_transmission"] = can_trans.run_number
 
 
 def _set_direct_files(job_request: JobRequest, sans_files: list[SansFileData]) -> None:
     direct_file = _find_direct_file(sans_files=sans_files)
-    logger.info("LOQ direct files found %s", direct_file)
+    logger.info("%s direct files found %s", job_request.instrument, direct_file)
     if direct_file is not None:
         if "scatter_transmission" in job_request.additional_values:
             job_request.additional_values["scatter_direct"] = direct_file.run_number
         if "can_scatter" in job_request.additional_values and "can_transmission" in job_request.additional_values:
             job_request.additional_values["can_direct"] = direct_file.run_number
 
 
-class LoqFindFiles(Rule[bool]):
+class CheckIfScatterSANS(Rule[bool]):
+    def __init__(self, value: bool):
+        super().__init__(value)
+        self.should_be_first = True
+
+    def verify(self, job_request: JobRequest) -> None:
+        if not job_request.experiment_title.endswith("_SANS/TRANS") and not job_request.experiment_title.endswith(
+            "_SANS"
+        ):
+            job_request.will_reduce = False
+            logger.error("Not a scatter run. Does not have _SANS or _SANS/TRANS at the end of the experiment title.")
+            return
+        # If it is a direct fix, sans or trans, it should fail, which is why hard coded TRANS as we want to check
+        # part of the logic not all.
+        if _is_sample_direct_file(
+            SansFileData(title=job_request.experiment_title, type="TRANS", run_number=str(job_request.run_number))
+        ):
+            job_request.will_reduce = False
+            logger.error(
+                "If it is a scatter, contains empty or direct in the title and is assumed to be a scatter "
+                "for an empty can run."
+            )
+            return
+        if "{" not in job_request.experiment_title and "}" not in job_request.experiment_title:
+            job_request.will_reduce = False
+            logger.error("If it is a scatter, contains {} in format {x}_{y}_SANS/TRANS. or {x}_SANS/TRANS.")
```

Comment:
I also think this is not very clear. These logs feel more like comments, but are still not very clear. And about the log level.